config/tizen/packaging/iotjs.spec

```diff
@@ -0,0 +1,105 @@
+Name: iotjs
+Version: 1.0.0
+Release: 0
+Summary: Platform for Internet of Things with JavaScript
+Group: Network & Connectivity
+License: Apache-2.0
+URL: https://www.iotjs.net/
+Source:     %{name}-%{version}.tar.gz
+Source1:    %{name}.pc.in
+Source1001: %{name}.manifest
+
+BuildRequires: python
+BuildRequires: cmake
+BuildRequires: glibc-static
+BuildRequires: aul
+BuildRequires: pkgconfig(appcore-agent)
+BuildRequires: pkgconfig(capi-appfw-service-application)
+BuildRequires: pkgconfig(capi-appfw-app-common)
+BuildRequires: pkgconfig(capi-appfw-package-manager)
+BuildRequires: pkgconfig(capi-appfw-application)
+BuildRequires: pkgconfig(dlog)
+#for https
+BuildRequires:  openssl-devel
+BuildRequires:  libcurl-devel
+
+Requires(postun): /sbin/ldconfig
+Requires(post): /sbin/ldconfig
+
+%description
+Platform for Internet of Things with JavaScript
+
+# default is RELEASE mode.
+# If DEBUG mode is needed, please use tizen_build_devel_mode
+%define RELEASE False
+# For Example
+%if %{RELEASE} == "True"
+%define build_mode release
+%else
+%define build_mode debug
+%endif
+
+# Default values to be eventually overiden BEFORE or as gbs params:
+%{!?RELEASE: %define RELEASE 0}
+
+%package service
+Summary: Development files for %{name}
+Group: Network & Connectivity/Service
+Requires: %{name} = %{version}-%{release}
+
+%description service
+The %{name}-service package contains service iotjs files for
+developing applications that use %{name}.
+
+%package devel
+Summary: Header files for %{name}
+Group: Network & Connectivity/Service
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+Development libraries for %{name}
+
+%prep
+%setup -q -c
+chmod g-w %_sourcedir/*
+cat LICENSE
+cp %{SOURCE1001} .
+touch abc.txt
+
+%build
+./tools/build.py --clean --buildtype=%{build_mode} --target-arch=arm \
+ --target-os=tizen --target-board=artik10 \
+ --no-init-submodule --no-parallel-build --no-check-test
```

Comment:
Yes, I'm sure. GBS can not make update with git submodules on build.py is running. That's why .git folder removed. You can try its build with this file(copy this on your $HOME, ~/) and 
make a script copy (./config/tizen/.gbsbuild.sh .) on the working.
You can see this script pull the submodule before removing .git folder. 

https://gist.githubusercontent.com/sumin-lim/fcbfdae32f76954e462dd615f8d93e91/raw/677ae88a23f51b9c89ba0ec234843f3e89b41508/.gbs.conf